### PR TITLE
[Test fix] Test failure: tracing_eventsource._eventpipeandetw_eventpipeandetw_eventpipeandetw_/_eventpipeandetw_eventpipeandetw_eventpipeandetw_cmd

### LINKED ISSUE #19302: Test failure: tracing_eventsource._eventpipeandetw_eventpipeandetw_eventpipeandetw_/_eventpipeandetw_eventpipeandetw_eventpipeandetw_cmd
Opened on behalf of @Sunny-pu

The test `tracing_eventsource._eventpipeandetw_eventpipeandetw_eventpipeandetw_/_eventpipeandetw_eventpipeandetw_eventpipeandetw_cmd` has failed.

    Unhandled Exception: System.InvalidOperationException: TdhEnumerateProviders failed.\r
       at Microsoft.Diagnostics.Tracing.Session.TraceEventSession.get_ProviderNameToGuid()\r
       at Microsoft.Diagnostics.Tracing.Session.TraceEventProviders.GetProviderGuidByName(String name)\r
       at Microsoft.Diagnostics.Tracing.Session.TraceEventSession.EnableProvider(String providerName, TraceEventLevel providerLevel, UInt64 matchAnyKeywords, TraceEventProviderOptions options)\r
       at Tracing.Tests.EventPipeAndEtw.EnableETW(EventSource eventSource, EventKeywords keywords, String outputFile) in E:\\A\\_work\\138\\s\\tests\\src\\tracing\\eventsource\\eventpipeandetw\\EventPipeAndEtw.cs:line 113\r
       at Tracing.Tests.EventPipeAndEtw.RoundOne(String[] args) in E:\\A\\_work\\138\\s\\tests\\src\\tracing\\eventsource\\eventpipeandetw\\EventPipeAndEtw.cs:line 144\r
       at Tracing.Tests.EventPipeAndEtw.Main(String[] args) in E:\\A\\_work\\138\\s\\tests\\src\\tracing\\eventsource\\eventpipeandetw\\EventPipeAndEtw.cs:line 287
    
    
    Return code:      1
    Raw output file:      C:\\work\\f2125dcf-0985-4d70-b7d0-00824a6edc84\\Work\\c8415983-ece0-46e7-b42d-256fbc794b63\\Unzip\\Reports\\tracing.eventsource\\eventpipeandetw\\eventpipeandetw\\eventpipeandetw.output.txt
    Raw output:
    BEGIN EXECUTION\r
            1 file(s) copied.\r
    \" C:\\work\\f2125dcf-0985-4d70-b7d0-00824a6edc84\\Payload\\crossgen.exe\" /Platform_Assemblies_Paths C:\\work\\f2125dcf-0985-4d70-b7d0-00824a6edc84\\Payload;C:\\work\\f2125dcf-0985-4d70-b7d0-00824a6edc84\\Work\\c8415983-ece0-46e7-b42d-256fbc794b63\\Unzip\\eventpipeandetw\\eventpipeandetw\\IL;C:\\work\\f2125dcf-0985-4d70-b7d0-00824a6edc84\\Work\\c8415983-ece0-46e7-b42d-256fbc794b63\\Unzip\\eventpipeandetw\\eventpipeandetw /in eventpipeandetw.org /out eventpipeandetw.exe\r
    Microsoft (R) CoreCLR Native Image Generator - Version 4.5.30319.0\r
    Copyright (c) Microsoft Corporation.  All rights reserved.\r
    \r
    Native image eventpipeandetw.exe generated successfully.\r
     \"C:\\work\\f2125dcf-0985-4d70-b7d0-00824a6edc84\\Payload\\corerun.exe\" eventpipeandetw.exe \r
    \tStart: Enable EventPipe.\r
    \tEnd: Enable EventPipe.
    \r
    \tStart: Enable ETW.\r
    Expected: 100\r
    Actual: -532462766\r
    END EXECUTION - FAILED\r
    FAILED\r
    Test Harness Exitcode is : 1\r
    
    To run the test:
    > set CORE_ROOT=C:\\work\\f2125dcf-0985-4d70-b7d0-00824a6edc84\\Payload
    > C:\\work\\f2125dcf-0985-4d70-b7d0-00824a6edc84\\Work\\c8415983-ece0-46e7-b42d-256fbc794b63\\Unzip\\eventpipeandetw\\eventpipeandetw\\eventpipeandetw.cmd
    \r
    Expected: True\r
    Actual:   False

        Stack Trace:

        
Build : 3.0 - 20180806.01 (Ready-To-Run Tests)
Failing configurations:
- Windows.10.Nano.Amd64
  - x64

Details: https://mc.dot.net/#/product/netcore/30/source/official~2Fcoreclr~2Fmaster~2F/type/test~2Ffunctional~2Fr2r~2Fcli~2F/build/20180806.01/workItem/tracing.eventsource.XUnitWrapper/analysis/xunit/tracing_eventsource._eventpipeandetw_eventpipeandetw_eventpipeandetw_~2F_eventpipeandetw_eventpipeandetw_eventpipeandetw_cmd

### PR DESCRIPTION
**Pull request by jorive**
- Fixes (by disabling) #19302, #22045, #22826
For more information: https://github.com/dotnet/coreclr/issues/19302#issuecomment-460344470

### CODE REVIEW

**Comment by jorive** [comment]
@dotnet-bot test this please

**Comment by jorive** [comment]
@dotnet-bot test this please

File: tests/src/tracing/eventsource/eventpipeandetw/eventpipeandetw.csproj [hunk 1]
```diff
@@ -16,7 +16,12 @@
 
     <!-- Test unsupported outside of windows -->
     <TestUnsupportedOutsideWindows>true</TestUnsupportedOutsideWindows>
-    <DisableProjectBuild Condition="'$(TargetsUnix)' == 'true'">true</DisableProjectBuild>
+
+    <!--
+      Test is blocking official build, disabling it for now.
+      For more information check: #19302, #22045, #22826.
```
Comment (vancem) [inline tests/src/tracing/eventsource/eventpipeandetw/eventpipeandetw.csproj]: @jorive  Please make a single issue that then refers to all the other stuff.   This gives you a place to put a proper milestone and any information you collect about it in a way that won't get lost.   It also makes it super clear which update should re-enable this test.    Put that issue number here.

**Comment by jorive** [comment]
@vancem I have created an issue, and I'll add the related issue after the legs are finally done running, to avoid re-triggering all of them all over again.

**Pull request merged**